Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: test
+
+test: database-v1
+	@go test -v -race ./...
+
+database-v1:
```

Comment:
It's just two files instead of one; the extracted database is already isolated. I would just add `database-v2.tar.xz` to `.gitignore` and leave it at that for now. I wouldn't object to making a `testdata/` directory and putting both in there, but I don't think it's necessary until we have more than two things.